src/TheGuardianIn2020Banner/canRender.ts

```diff
@@ -0,0 +1,8 @@
+import type { BrazeMessageProps } from './index';
+
+export const COMPONENT_NAME = 'TheGuardianIn2020Banner';
+
+export const canRender = (brazeMessageProps: BrazeMessageProps): boolean => {
+    const { header, body } = brazeMessageProps;
+    return Boolean(header && body);
```

Comment:
Oh ha I made the same suggestion to rewrite to this above. I promise I didn't see this first :-)